src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Managed.cs

```diff
@@ -71,6 +75,8 @@ partial void InitializePartial()
 
 		private void HookScrollEvents(ScrollViewer sv)
 		{
+			UnhookScrollEvents(sv);
```

Comment:
As there a automated test that we can have to validate this?

Reply:
Yes, definitely, was planning to add it 👍